lib/screens/category_list_view.dart

```diff
@@ -0,0 +1,183 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:maui/games/single_game.dart';
+import 'package:maui/screens/select_opponent_screen.dart';
+import 'package:maui/repos/notif_repo.dart';
+import 'package:badge/badge.dart';
+import 'package:maui/loca.dart';
+import 'package:maui/components/gameaudio.dart';
+
+import '../db/entity/category.dart';
+import '../repos/activity_template_repo.dart';
+import '../repos/category_repo.dart';
+import 'sub_category_list_view.dart';
+
+class CategoryListView extends StatefulWidget {
+  const CategoryListView({Key key}) : super(key: key);
+
+  @override
+  GameListViewState createState() {
+    return new GameListViewState();
+  }
+
+  showModes(BuildContext context, String game) {
+    Navigator.of(context).pushNamed('/categories/$game');
+  }
+}
+
+class GameListViewState extends State<CategoryListView> {
+  Map<String, int> _notifs = Map<String, int>();
+ List<Category> _datacategory = new List<Category>();
+  var _datatemplate;
+  var _categorydata;
+ 
+  @override
+  void initState() {
+    super.initState();
+    _initData();
+  }
+
+  void _initData() async {
+    var notifs = await NotifRepo().getNotifCountByType();
```

Comment:
We dont need notifs here. This is for displaying notifications on the game icon. When copying and pasting code, please make sure you understand what you are copying